gst/nnstreamer/tensor_query/tensor_query_common.c

```diff
@@ -884,16 +925,35 @@ accept_socket_async_cb (GObject * source, GAsyncResult * result,
 }
 
 /**
- * @brief set server source and sink tensors config.
+ * @brief return newly allocated string.
+ */
+static char *
+_strdup (const char *str)
```

Comment:
why do you add this function? (IMO, simply use g_strdup() )

Reply:
I tried to avoid using glib. I think it would be better to remove glib dependence at once later. 
Updated.